LibTrixi.jl/src/api_c.jl

```diff
@@ -461,6 +461,53 @@ trixi_load_primitive_vars_cfptr() =
     @cfunction(trixi_load_primitive_vars, Cvoid, (Cint, Cint, Ptr{Cdouble}))
 
 
+"""
+    trixi_store_in_database(data::Ptr{Cdouble}, size::Cint, index::Cint,
+                            simstate_handle::Cint)::Cvoid
```

Comment:
I am not 100% sure I like the name "database", since it has a very specific meaning in computer science. Also, it does not really "store" any data but only a reference to it, right? To me, the current name implies that data is actually saved into some Trixi-allocated containers.

Alas, I do not have a better name, but maybe you have another idea?

Reply:
What about registry?

Reply:
Why not? Although registry has a specific meaning in Julia as well... what about "data registry" (too long?) or "data bridge" (to emphasize its purpose) or "data pool" (like cache, but cache is already with a strong meaning in the Trixi.jl world)?

Reply:
I just saw your comment that you renamed it to "data registry", so that's that 😊